src/js/_enqueues/admin/common.js

```diff
@@ -2245,3 +2245,105 @@ $( function( $ ) {
+                tooltipButton.classList.add('tooltip-button');
+                tooltipButton.innerHTML =
+                    '<span class="dashicons dashicons-editor-help"></span>';
+                tooltipContainer.appendChild(tooltipButton);
+
+                var tooltipContent = document.createElement('div');
+                tooltipContent.classList.add('tooltip-content');
+
+				tooltipContent.innerHTML =
+					'<div class="tooltip-arrow"></div>' + '<p>' + tooltipText + '</p>';
+				tooltipContainer.appendChild(tooltipContent);
+
+                // Change event from click to hover
+                tooltipContainer.addEventListener('mouseenter', function () {
+                    // Show the tooltip content on hover
+                    tooltipContent.style.display = 'block';
+                });
+
+                tooltipContainer.addEventListener('mouseleave', function () {
+                    // Hide the tooltip content on mouse leave
+                    tooltipContent.style.display = 'none';
+                });
+
+                // Set cursor style to 'help' when hovering over the tooltip button
+                tooltipButton.style.cursor = 'help';
+
+                // Add keydown event listener to the tooltip button
+                tooltipButton.addEventListener('keydown', function (event) {
+                    // Check if the Enter key is pressed
+                    if (event.key === 'Enter') {
+                        // Toggle the display of the tooltip content
+                        tooltipContent.style.display =
+                            tooltipContent.style.display === 'block' ? 'none' : 'block';
+                    }
+                });
+
+                document.addEventListener('keydown', function (event) {
+                    if (
+                        event.key === 'Escape' &&
+                        tooltipContent.style.display === 'block'
+                    ) {
+                        // Hide the tooltip on Escape key press
+                        tooltipContent.style.display = 'none';
+                        tooltipButton.focus();
+                    }
+                });
+
+                document.body.addEventListener('click', function (event) {
+                    // Check if the clicked element is not within the tooltip container
+                    if (
+                        !tooltipContent.contains(event.target) &&
+                        !tooltipButton.contains(event.target)
+                    ) {
+                        tooltipContent.style.display = 'none';
+                    }
+                });
+            }
+        });
+
+        // Tooltips have been initialized
+        tooltipsInitialized = true;
+    }
+
+    // Initialize tooltips on DOMContentLoaded
+    $(document).ready(function () {
+        initializeTooltips();
+    });
+
+    // Reinitialize tooltips after AJAX events
+    $(document).ajaxComplete(function () {
+        initializeTooltips();
+    });
```

Comment:
It will run after any AJAX request do we need to limit it so it will run after some specific action? 🤔

Reply:
Thanks for the catch @mukeshpanchal27. Well, yeah I do not think initializing it on every AJAX will be a great idea, so I have modified it to only initialize it when the DOM is ready. Removed the unnecessary parts.